### ISSUE TITLE
Display notice when draft has been deleted before

### ISSUE THREAD

**Issue by NovemLinguae** [WPAFC/afch-rewrite#160]
Similar to how a notice is given when the mainspace article has been deleted before, consider also displaying a notice for when the draft has been deleted before.

Example: https://en.wikipedia.org/wiki/Draft:Jack_Maguire_(violinist). Has both mainspace and draft deletes.

Consider filtering out G13's from draft deletion notices, since those aren't anything bad or alarming.

Consider filtering out R2's from mainspace deletion notices, since those aren't anything bad or alarming.

![image](https://user-images.githubusercontent.com/79697282/114299862-623c4b80-9a72-11eb-8d8d-cacc268b2fff.png)

**Comment by Ashish-100-tiwari** [comment]
@NovemLinguae is this issue is still open ? if yes , can I work on it .

**Comment by Ashish-100-tiwari** [comment]
is this issue still open ? if yes , can I work on it

**Comment by enterprisey** [comment]
Yes. You'll need to figure out where warnings get displayed, and add a new warning that checks the deletion log for the draft. It should be similar to the existing code. Honestly we should probably change the existing code to take a page name as a parameter, because otherwise we'd be copying and pasting the function and I don't think that's a good option.
